contrib/inventory/gce.py

```diff
@@ -272,13 +275,17 @@ def get_gce_driver(self):
         # exists.
         secrets_path = self.config.get('gce', 'libcloud_secrets')
         secrets_found = False
-        try:
-            import secrets
-            args = list(getattr(secrets, 'GCE_PARAMS', []))
-            kwargs = getattr(secrets, 'GCE_KEYWORD_PARAMS', {})
-            secrets_found = True
-        except:
-            pass
+
+        # check if secrets exist in the current directory, to avoid
+        # loading the python 3 module 'secrets'
+        if os.path.exists('secrets.py'):
```

Comment:
Hi Erjohnso,

Because there is a builtin Python 3 module named 'secrets' (check https://docs.python.org/3/library/secrets.html ), the wrong module is imported in the case of Python 3.

What do you suggest?